### PR TITLE
Servers object storage

### PR DESCRIPTION
**Pull request by midlik**
# Description

Support input from Google Cloud Storage (gs://) for ModelServer and VolumeServer.

## Actions

- [x] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [x] Updated headers of modified files
- [x] Added my name to `package.json`'s `contributors`
- [ ] (Optional but encouraged) Improved documentation in `docs`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -154,6 +154,7 @@
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
+    "@google-cloud/storage": "^7.14.0",
```
[the remark this answers is not in the document]
Comment (midlik) [reply package.json]: OK, i changed it.

File: src/servers/model/utils/fetch-retry.ts [hunk 2]
```diff
@@ -31,3 +34,15 @@ export async function fetchRetry(url: string, timeout: number, retryCount: numbe
...
+export async function extendedFetch(url: string, init?: fetch.RequestInit): Promise<Response> {
+    if (url.startsWith('gs://')) return fetch_GS(url, init);
+    else return fetch(url, init);
+}
+
+async function fetch_GS(url: string, init?: fetch.RequestInit): Promise<Response> {
+    if (init?.signal?.aborted) throw new AbortError('The user aborted a request.');
+    const data = await downloadGs(url);
+    return new Response(data, init);
+}
```
Comment (dsehnal) [inline src/servers/model/utils/fetch-retry.ts:48]: If there is no await, no need for async.

```suggestion
export function wrapFetch(url: string, init?: fetch.RequestInit): Promise<Response> {
    if (url.startsWith('gs://')) return fetchGS(url, init);
    else return fetch(url, init);
}

async function fetchGS(url: string, init?: fetch.RequestInit): Promise<Response> {
    if (init?.signal?.aborted) throw new AbortError('The user aborted a request.');
    const data = await downloadGs(url);
    return new Response(data, init);
}
```

**Review by dsehnal — APPROVED** [review]
LGTM, just a small tweak

File: src/servers/model/utils/fetch-retry.ts (hunk 2 above)
Comment (midlik) [reply src/servers/model/utils/fetch-retry.ts]: I always get a bit confused by functions that are not async but return a promise :D
But you're right.
I changed it.

**Pull request merged**